pkg/platform/packet/packet.go

```diff
@@ -528,3 +532,22 @@ func checkResFormat(reservationIDs map[string]string, name, errorPrefix, resPref
 
 	return diagnostics
 }
+
+func (wp *workerPool) withInheritedValuesFromConfig(c *config) workerPool {
+	// Preserve user provided values for worker pool.
+	workerpool := *wp
+
+	if len(wp.SSHPubKeys) == 0 {
+		workerpool.SSHPubKeys = c.SSHPubKeys
+	}
+
+	if len(wp.CLCSnippets) == 0 {
```

Comment:
I don't think CLC snippets should be inherited, as then there is no way to disable it for workers.

Reply:
reworked the PR, not inheriting.